zrml/swaps/src/lib.rs

```diff
@@ -1146,7 +1143,8 @@ mod pallet {
             amount: BalanceOf<T>,
         ) -> DispatchResult {
             let shares_id = Self::pool_shares_id(pool_id);
-            T::Shares::deposit(shares_id, to, amount)
+            T::Shares::deposit(shares_id, to, amount)?;
+            Ok(())
```

Comment:
This change is unnecessary, since it does the same as `T::Shares::deposit(shares_id, to, amount)` (return `Ok(())` or the error)

Reply:
For a while there was a `deposit_event` call between those, and I forgot to put things back to the way they were 🙈